### PR TITLE
Mention marker symbols

### PR DESCRIPTION
**Pull request by mschauer**

### CODE REVIEW

**Comment by ffreyer** [comment]
This should also mention that any `Char` works. `available_marker_symbols` only contains the named (i.e. Symbol) markers

File: docs/make.jl [hunk 1]
```diff
@@ -248,7 +248,7 @@ const plot_attr_desc = Dict(
     :levels => "Integer. Number of levels for a `contour`-type plot.",
     :linestyle => "Symbol. Style of the line (for `line` and `linesegments` plots). Available styles are `:dash`, `:dot`, `:dashdot`, and `:dashdotdot`. You can also supply an array describing the length of each gap/fill.",
     :linewidth => "Number. Width of the line in `line` and `linesegments` plots.",
-    :marker => "Symbol, Shape, or AbstractVector.",
+    :marker => "Symbol, Shape, or AbstractVector. Call `available_marker_symbols`() to see which ones.",
```
Comment (mschauer) [inline docs/make.jl:251]:
```suggestion
    :marker => "Symbol, Char, Shape, or AbstractVector. Call `available_marker_symbols`() to see which ones.",
```

**Comment by mschauer** [comment]
Good to go? This is meant to be low effort :-) , just didn't want the answer to the question I asked on Slack go to waste.

**Comment by SimonDanisch** [comment]
Yeah, thanks a lot :)

**Comment by kragol** [comment]
This is a pretty useful addition to the docs. 

Unfortunately, `available_marker_symbols` is not exported at the moment. Could this be added to the PR? (Doesn't seem worth creating a new one).

**Comment by mschauer** [comment]
Exporting it or adding a remark?

**Pull request merged**